src/clojure/playasophy/wonderdome/mode/flicker.clj

```diff
@@ -1,18 +1,19 @@
 (ns playasophy.wonderdome.mode.flicker
   (:require
     [playasophy.wonderdome.mode.core :as mode]
-    [playasophy.wonderdome.util.color :as color]))
+    [playasophy.wonderdome.util.color :as color]
+    [playasophy.wonderdome.util.control :as control]))
 
 
 (def ^:const ^:private hue-range 0.137)
 (def ^:const ^:private hue-delta 0.05)
 
-(def ^:const ^:private brightness-rate 0.027)
+(def ^:const ^:private brightness-rate 0.05)
 
 ; TODO: 'speed' is more like 'delay'
-(def ^:const ^:private min-speed 10)
+(def ^:const ^:private min-speed 5)
 (def ^:const ^:private max-speed 3500)
-(def ^:const ^:private speed-rate 100)
+(def ^:const ^:private speed-rate -200)
```

Comment:
Yeah. Probably better to refactor the variable to mean the inverse of what it does currently, but that was more work. This was quick and dirty.